### ISSUE TITLE
Adding a transparent option to color pickers

### ISSUE THREAD

**Issue by claudiaareneee** [iTwin/itwinjs-core#3197]
**Is your feature request related to a problem? Please describe.**
For the color pickers, it would be helpful to include a stylized transparent option. Currently when adding a transparent color to the color picker options, there is no indication that the option is transparent

**Describe the solution you'd like**
Adding a prop to the ColorPickerButton / ColorPickerPopup for including a transparent option that's stylized to meet the UI standard for transparent color swatches

**Describe alternatives you've considered**
The transparent color could also just be included in the default colors of the color pickers

**Additional context**
Current ColorPickerButton - the selected option is transparent, but there's no styling to show that
![image](https://user-images.githubusercontent.com/25939215/153615818-318c13c6-7fd3-46b1-85ae-218c5ff22795.png)

Design standards
![image](https://user-images.githubusercontent.com/25939215/153616051-6e858f69-40b0-4d88-a6b5-e26d8b602e88.png)
[iTwin UI color picker](https://ux.bentley.com/itwin/components/color-picker.html#gallery-3)

Current versions:
```
"@itwin/imodel-components-react": "^3.0.0"
"@itwin/core-common": "^3.0.0"
"@itwin/core-frontend": "^3.0.0"
```

**Comment by NancyMcCallB** [comment]
We can change the styling for the button, but perhaps you'd get faster results by using the [color picker](https://itwin.github.io/iTwinUI-react/?path=/story/core-colorpicker--advanced) in itwinui-react, which this component wraps.

**Comment by claudiaareneee** [comment]
Thanks! I'll take a look

**Comment by mayank99** [comment]
@NancyMcCallB Are you sure this component wraps iTwinUI's ColorPicker? I was looking at [`ui/src/imodel-components-react/color/`](https://github.com/iTwin/itwinjs-core/blob/master/ui/imodel-components-react/src/imodel-components-react/color/) and I don't see it referencing iTwinUI.

**Comment by mayank99** [comment]
@NancyMcCallB Are you sure this component wraps iTwinUI's ColorPicker? I was looking at [`ui/src/imodel-components-react/color/ColorPickerButton`](https://github.com/iTwin/itwinjs-core/blob/master/ui/imodel-components-react/src/imodel-components-react/color/ColorPickerButton.tsx) and I don't see it referencing iTwinUI.

However, if a component wraps iTwinUI, it might help to add a link in the JSDoc. Also, it might help if we add a link to iTwinUI in the issue templates.

**Comment by NancyMcCallB** [comment]
The button does not, but the actual color picker it opens does use iTwinui-react.

**Comment by mayank99** [comment]
Ah, makes sense, thanks for clarifying. Then we will try to handle this in iTwinUI.

**Comment by mayank99** [comment]
🎉 This is now available by default in `@itwin/itwinui-react@1.32.0`. Any swatch with transparency will have a checkered background:
![image](https://user-images.githubusercontent.com/9084735/155752408-51690e9d-7065-4fa0-87fa-87399afbb877.png)
